Src/EB/AMReX_EB2_Level.H

```diff
@@ -479,6 +495,34 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
         }
 
         ParallelAllReduce::Sum<int>({nsmallcells,nmulticuts}, ParallelContext::CommunicatorSub());
+        if (plt_multiple_cuts && nmulticuts > 0)
+        {
+            amrex::Print() << "Total number of multicuts = " << nmulticuts << "\n";
+            amrex::Print() << "plotting multicut locations..." << "\n";
+
+            #if (AMREX_SPACEDIM == 3)
+                if (!multicut_face[0].empty())
+                {
+                    WriteSingleLevelPlotfile("plt.x", multicut_face[0], {"multicut_fcx"}, geom, 0.0, 0);
+                }
+                if (!multicut_face[1].empty())
+                {
+                    WriteSingleLevelPlotfile("plt.y", multicut_face[1], {"multicut_fcy"}, geom, 0.0, 0);
+                }
+                if (!multicut_face[2].empty())
+                {
+                    WriteSingleLevelPlotfile("plt.z", multicut_face[2], {"multicut_fcz"}, geom, 0.0, 0);
+                }
+            #elif (AMREX_SPACEDIM == 2)
+                if (!multicut_face[0].empty())
+                {
+                    WriteSingleLevelPlotfile("plt.x", multicut_face[0], {"multicut_fcx"}, geom, 0.0, 0);
+                }
+            #endif
+
+            amrex::Abort("amrex::EB2::build_faces: more than 2 cuts not supported");
```

Comment:
change the abort message because this is no longer being called from build_faces